routes/webhooks/sffd.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const HttpStatus = require('http-status-codes');
+
+const models = require('../../models');
+
+const router = express.Router();
+
+const UNIT_SFFD_REGEX = /^\d+$/;
+const UNIT_AM_REGEX = /^AM\d+$/;
+const UNIT_KM_REGEX = /^KM\d+$/;
+
+router.post('/cad', async (req, res) => {
+  const data = req.body;
+  if (!Array.isArray(data)) {
+    return;
+  }
+  await models.sequelize.transaction(async (transaction) => {
+    const superUser = await models.User.findOne({
+      where: { email: 'batsadmin@c4sf.me' },
+      transaction,
+    });
+    const sffd = await models.Organization.findOne({
+      where: { name: 'San Francisco Fire Department' },
```

Comment:
@billhoke should we add a stable identifier column to Organization, separate from the auto generated UUIDs and the display name, to prevent this from breaking if the display name changes? I.e. "sffd", "amr", "king"...

Reply:
alternatively, we generate well-defined UUIDs for these orgs, and hard-code those values...?

Reply:
@francisli - I'm not sure what will break if an organization name changes. Is there a dependency? I'm drawing a blank on that one ...

Reply:
@billhoke well, I'm just saying that if the name of the org changes in the database, this code will break. Typically, I prefer not to do queries on a "display" name string that someone might want edited. For example, if "San Francisco Fire Department" gets changed to "City and County of San Francisco Fire Department" (which is the official name in the NEMSIS registry). I often add another slug/handle unique column for an id that is easy to reference and can be stable (I.e. "sffd") even if the display name is edited.